src/infrastructure/models/bom_version.rs

```diff
@@ -25,6 +27,39 @@ pub struct BomVersion {
     pub id: Uuid,
     pub bom_id: Uuid,
     pub version: i32,
-    pub changes: Value,
+    pub changes: DbBomVersionChanges,
```

Comment:
`serde::Value` is quite generic, someone could pass any kind of "Value", creating a type that implements `FromSql` and `ToSql` gives us better type-safety on the column / field